src/data/tableColumns/tableWithTabs.js

```diff
@@ -0,0 +1,65 @@
+export default [
+    {
+        "name": "id",
+        "label": "id",
+        "field": "id",
+        "align": "left",
+        "sortable": true
+    },
+    {
+        "name": "name",
+        "label": "name",
+        "field": "name",
+        "align": "left",
+        "sortable": true
```

Comment:
sortable true and align left should also be default that don't need to be repeated

Reply:
done